crates/bevy_animation/src/animation_curves.rs

```diff
@@ -844,11 +885,20 @@ where
             None => self.blend_register = Some((value_to_blend, weight_to_blend)),
             Some((ref mut current_value, ref mut current_weight)) => {
                 *current_weight += weight_to_blend;
-                *current_value = A::interpolate(
-                    current_value,
-                    &value_to_blend,
-                    weight_to_blend / *current_weight,
-                );
+
+                if additive {
+                    *current_value = A::blend(iter::once(BlendInput {
```

Comment:
This currently just ignores the existing value in the blend register, which doesn't seem right. I think that for the present implementation of `Animatable::blend`, the right thing to do would be to use an iterator of length two with two blend inputs, one for the current value and one for the incoming one, with both inputs set to additive.

(Probably we could also just change the definition of `Animatable::blend` to support the blend stacks more directly in the future as well; I don't think we use it for anything other than what's happening here.)

Reply:
Oops! I feel silly now :) Fixed.